test/integration/full/region/region-pass.js

```diff
@@ -13,12 +13,10 @@ describe('region pass test', function() {
 	});
 
 	describe('passes', function() {
-		it('should find one', function() {
-			assert.lengthOf(results.passes[0].nodes, 1);
```

Comment:
Lets test the number of elements we expect from the fixture, just to make sure.

Reply:
The problem I had with that is that it was inconsistent. You can see from this test that the [number failed in the unix test](https://app.circleci.com/jobs/github/dequelabs/axe-core/11652) but will pass in localhost.

![image](https://user-images.githubusercontent.com/2433219/73095189-42396000-3e9f-11ea-902a-7f7f3087fc95.png)

<img width="548" alt="Screen Shot 2020-01-24 at 11 45 55 AM" src="https://user-images.githubusercontent.com/2433219/73095105-20d87400-3e9f-11ea-9565-2564b2e0e5e0.png">